Release 14.1.4

camscale opened the pull request:
## 14.1.4 (11/15/2023)

* Fixed incorrect permissions when opening X11 listener. [#34617](https://github.com/gravitational/teleport/pull/34617)
* Environment values can not be overridden from the .tsh/environment file, only unique keys will be inserted into the environment. [#34626](https://github.com/gravitational/teleport/pull/34626)
* Access lists now respect user locking. [#34620](https://github.com/gravitational/teleport/pull/34620)
* Respect explicit deny rules for Access Requests. [#34600](https://github.com/gravitational/teleport/pull/34600)
* Added Teleport Access Graph integration. [#34569](https://github.com/gravitational/teleport/pull/34569)
* Fixed cleanup of unused GCP KMS keys. [#34468](https://github.com/gravitational/teleport/pull/34468)
* Added list view option to the unified resources page. [#34466](https://github.com/gravitational/teleport/pull/34466)
* Fixed duplicate entries in resources view when updating nodename [#34236](https://github.com/gravitational/teleport/issues/34236) [#34453](https://github.com/gravitational/teleport/pull/34453)
* Allow configuring `cluster_networking_config` and `cluster_auth_preference` via `--bootstrap`. [#34445](https://github.com/gravitational/teleport/pull/34445)
* Fixed `tsh logout` with broken key directory. [#34435](https://github.com/gravitational/teleport/pull/34435)
* Added binary formatted parameters as base64 encoded strings to PostgreSQL Statement Bind audit log events. [#34432](https://github.com/gravitational/teleport/pull/34432)
* The operator reuses its connection to Teleport. Reduces CPU usage, logs, and fixes a memory leak. [#34425](https://github.com/gravitational/teleport/pull/34425)
* Updated the code signing certificate for Windows artifacts. [#34377](https://github.com/gravitational/teleport/pull/34377)
* Added IAM Authentication support for Amazon MemoryDB Access. [#34348](https://github.com/gravitational/teleport/pull/34348)
* Split large desktop recordings into multiple files during export. [#34319](https://github.com/gravitational/teleport/pull/34319)
* Allow setting server labels from tctl. [#34137](https://github.com/gravitational/teleport/pull/34137)

---

### Ignored in changelog

#### Docs
* [v14] Link to the Usage/Billing page in the FAQ [#34583](https://github.com/gravitational/teleport/pull/34583)
* [v14] docs: Mark AWS GSLB deployment as Enterprise only [#34623](https://github.com/gravitational/teleport/pull/34623)
* [v14] docs: correct fips image name reference [#34595](https://github.com/gravitational/teleport/pull/34595)
* [v14] Support ClusterNetworkingConfig and AuthPreference in --apply-on-startup  [#34573](https://github.com/gravitational/teleport/pull/34573)
* Added 11/13 Upcoming Releases Update [#34553](https://github.com/gravitational/teleport/pull/34553)
* [v14] Connect My Computer docs: Add role permissions to prerequisites [#34521](https://github.com/gravitational/teleport/pull/34521)
* [v14] Machine ID: More rules to `gitlab` joining and add globby matching to… [#34464](https://github.com/gravitational/teleport/pull/34464)
* [v14] Update kubernetes.mdx [#34448](https://github.com/gravitational/teleport/pull/34448)
* [v14] Remove version warnings in Details boxes [#34429](https://github.com/gravitational/teleport/pull/34429)
* [v14] Fix default piv key slot to match comments and docs [#34414](https://github.com/gravitational/teleport/pull/34414)
* [v14] Add a Usage Reporting and Billing page [#34361](https://github.com/gravitational/teleport/pull/34361)
* [v14] Remove cloud only warning for jira plugin [#34301](https://github.com/gravitational/teleport/pull/34301)
* [v14] Create a guide to using Teleport with Spacelift [#34246](https://github.com/gravitational/teleport/pull/34246)
* [v14] Edit the teleport-cluster Helm guide [#34104](https://github.com/gravitational/teleport/pull/34104)
* [v14] Expand the Access Controls reference [#33967](https://github.com/gravitational/teleport/pull/33967)

#### e updates
* [v14] Bump e ref for new Cloud email invite UI [#34568](https://github.com/gravitational/teleport/pull/34568)
* Update e [#34473](https://github.com/gravitational/teleport/pull/34473)

#### Minor/trivial user-visible changes
* [v14] Supporting changes for Cloud email invites (#32439) [#34180](https://github.com/gravitational/teleport/pull/34180)

#### Testing change
* [v14] wait for nodes to register in 'TestIntegrations/DataTransfer' [#33628](https://github.com/gravitational/teleport/pull/33628)

#### No changelog
* [v14] Reword aws-oidc deploy service components [#34637](https://github.com/gravitational/teleport/pull/34637)
* [v14] Added release server publishing retry [#34641](https://github.com/gravitational/teleport/pull/34641)
* [v14] Grant the Okta service the right to write Users [#34607](https://github.com/gravitational/teleport/pull/34607)
* [v14] chore: Bump Buf to v1.28.0 [#34574](https://github.com/gravitational/teleport/pull/34574)
* [v14] Update e [#34563](https://github.com/gravitational/teleport/pull/34563)
* [v14] Web: Fix access lists overdue dates notification notes [#34555](https://github.com/gravitational/teleport/pull/34555)
* [v14] Fix data race in file descriptors [#34537](https://github.com/gravitational/teleport/pull/34537)
* [v14] Bump go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc from 0.45.0 to 0.46.0 in /examples [#34536](https://github.com/gravitational/teleport/pull/34536)
* [v14] Update e [#34533](https://github.com/gravitational/teleport/pull/34533)
* [v14] Implement temporary last step of Connect My Computer flow in Discover [#34522](https://github.com/gravitational/teleport/pull/34522)
* [v14] Fix label wrapping in Connect My Computer status view [#34520](https://github.com/gravitational/teleport/pull/34520)
* [v14] Poll for Connect My Computer node in the Discover flow [#34518](https://github.com/gravitational/teleport/pull/34518)
* [v14] Add Access Graph protobuf file [#34483](https://github.com/gravitational/teleport/pull/34483)
* [v14] Disabled kinds from filter menu based on access [#34480](https://github.com/gravitational/teleport/pull/34480)
* [v14] [Web] Unified Resources: Add skeleton loader to list view [#34478](https://github.com/gravitational/teleport/pull/34478)
* [v14] Update teleport command descriptions for consistency [#34477](https://github.com/gravitational/teleport/pull/34477)
* [v14] Bump github.com/sigstore/cosign/v2 to 2.2.1 in /integrations/kube-agent-updater [#34463](https://github.com/gravitational/teleport/pull/34463)
* [v14] [Web] EICE Discover: Extract `instanceId` from label [#34459](https://github.com/gravitational/teleport/pull/34459)
* [v14] Unified resources view improvements  [#34441](https://github.com/gravitational/teleport/pull/34441)
* [v14] desktop access: soften the disconnect error [#34421](https://github.com/gravitational/teleport/pull/34421)
* [v14] Update e [#34416](https://github.com/gravitational/teleport/pull/34416)
* [v14] Update working on tctl nodes add usage [#34407](https://github.com/gravitational/teleport/pull/34407)
* [v14] Introduce scaffolding for access list Slack notification. [#34404](https://github.com/gravitational/teleport/pull/34404)
* [v14] Update bloat check to match master [#34395](https://github.com/gravitational/teleport/pull/34395)
* [v14] Connect My Computer: Derive agent label from username in main process [#34385](https://github.com/gravitational/teleport/pull/34385)
* [v14] Fix floating error box in Unified View [#34374](https://github.com/gravitational/teleport/pull/34374)
* Release 14.1.3 [#34367](https://github.com/gravitational/teleport/pull/34367)
* [v14] Store original roles with user login state. [#34356](https://github.com/gravitational/teleport/pull/34356)
* [v14] oci: Parameterize Debian OCI Dockerfile for multiarch [#34188](https://github.com/gravitational/teleport/pull/34188)
* [v14] Enable Connect My Computer Discover tile & deep links in Connect [#33688](https://github.com/gravitational/teleport/pull/33688)

camscale committed:
Release 14.1.4

zmb3 commented on CHANGELOG.md:
```diff
@@ -1,5 +1,20 @@
+* Updated the code signing certificate for Windows artifacts. [#34377](https://github.com/gravitational/teleport/pull/34377)
+* Added IAM Authentication support for Amazon MemoryDB Access. [#34348](https://github.com/gravitational/teleport/pull/34348)
+* Split large desktop recordings into multiple files during export. [#34319](https://github.com/gravitational/teleport/pull/34319)
+* Exposed ServerInfo resource in tctl. [#34137](https://github.com/gravitational/teleport/pull/34137)
```
I don't expect anyone to know what this one means. How about
```suggestion
* Allow setting server labels from tctl. [#34137](https://github.com/gravitational/teleport/pull/34137)
```

camscale replied:
Sure. I'll update it. cc: @atburke

camscale committed:
update changelog of PR #34137

"Exposed ServerInfo resource in tctl"

becomes

" Allow setting server labels from tctl"

camscale committed:
Rebase on top of latest branch/v14 commits

This picks up three new PRs into v14.1.4

camscale committed:
More rebase on top of latest branch/v14 commits
